src/Explore/Views/map.twig

```diff
@@ -15,7 +15,7 @@
         <img class='dataIcon' id='crimeIcon' src='images/fingerprint.png' title='Crime Data by Severity' data-toggle='tooltip' data-placement='right'></img>
         <img class='dataIcon' id='schoolIcon' src='images/apple.png' title='Schools by Rating' data-toggle='tooltip' data-placement='right'></img>
         <!--<img class='dataIcon' id='foodIcon' src='images/food.png' title='Health Inspection Data' data-toggle='tooltip' data-placement='right'></img>-->
-        <img class='dataIcon' id='housingIcon' src='images/housingCost.png' title='Housing Market Data' data-toggle='tooltip' data-placement='right'></img>
+        <!--<img class='dataIcon' id='housingIcon' src='images/housingCost.png' title='Housing Market Data' data-toggle='tooltip' data-placement='right'></img>-->
```

Comment:
I'd prefer this be deleted rather than commented out. If we need it again we can look back at the commit history.